query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/QueryBuilder.java

```diff
@@ -402,6 +406,15 @@ public static Literal literal(@Nullable Object value) {
    */
   @NonNull
   public static Literal literal(@Nullable Object value, @NonNull CodecRegistry codecRegistry) {
+    if (value instanceof Murmur3Token) {
+      value = ((Murmur3Token) value).getValue();
```

Comment:
Nit: In theory you don't need to call `codecRegistry.codecFor(value)` for token values since you know the codecs to use already.